proxy/backend/src/main/java/org/apache/shardingsphere/proxy/backend/handler/admin/postgresql/executor/PostgreSQLSetCharsetExecutor.java

```diff
@@ -34,7 +34,7 @@ public final class PostgreSQLSetCharsetExecutor implements PostgreSQLSessionVari
     
     @Override
     public void handle(final ConnectionSession connectionSession, final String variableName, final String assignValue) {
-        String value = formatValue(assignValue.trim());
```

Comment:
Consider removing the method `formatValue` if it is unused.

Reply:
Ah! that was miss, it is removed now in the updated PR.